src/charts.js

```diff
@@ -39,15 +39,15 @@
     var LINE_STROKE_WIDTH = '1px';
     var POINT_STROKE_WIDTH = '1px';
 
-    if( !$element.jquery ){
```

Comment:
Probably want to rename the '$element' var to 'element' since it is not being used as a jQuery object any more